easybuild/scripts/bootstrap_eb.py

```diff
@@ -448,16 +511,18 @@ def main():
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 source_urls = [
-               'http://pypi.python.org/packages/source/e/easybuild-framework/',
-               'http://pypi.python.org/packages/source/e/easybuild-easyblocks/',
-               'http://pypi.python.org/packages/source/e/easybuild-easyconfigs/',
-              ]
+    'https://pypi.python.org/packages/source/v/vsc-base/',
```

Comment:
i would make these also templates, so it's easier to support download from github branch in a next PR :smile: